magento/magento2
# Clean up ruleset mixins at less file luma theme

### Description (*)

Update for luma theme only. Previously fixed in blank theme at ticket https://github.com/magento/magento2/pull/35101
Clean up ruleset mixins for luma theme (less)
=> Better approach
Selectors in referenced files expect that they have to be extended in OTHER NOT REFERENCED files. Don't use extend in the same referenced file.
Grunt doesn't compile extend selectors even without wrappers, such as: media-common or media mixins.
Use mixin instead of extend in the referenced file.

Current: &:extend(.abs-account-title all);
Better: @abs-account-title(); => mixins

Fix calc work at datepicker input

### Related Pull Requests

https://github.com/magento/magento2/pull/35101
### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. Compile styles via default deploying (setup:static-content:deploy) and via Grunt.
2. Config theme to luma -> Look at block titles of My Account (they should look the same after all build processes).
3. Enable show DOB of customer field in backend4. 
4. Go to frontend to visually changes5. 

### Questions or comments


### Contribution checklist (*)
